### DISCUSSION TITLE
Prep for release 1.3.0

### DISCUSSION THREAD

**Discussion by pwinckles** [performancecopilot/parfait#128] [General]
@natoscott @tallpsmith Thoughts on getting a new release out?

Changes since the last release

1. Fix `PcpMmvWriter` memory leaks https://github.com/performancecopilot/parfait/pull/121 and https://github.com/performancecopilot/parfait/pull/127
2. Fix `QuiescentRegistryListener` leak https://github.com/performancecopilot/parfait/pull/122
3. Fix parfait-agent packaging https://github.com/performancecopilot/parfait/pull/126
4. General dependency upgrades https://github.com/performancecopilot/parfait/pull/124

**Comment by natoscott** [comment]
@pwinckles +1 from me, sooner the better.

**Reply by pwinckles** [reply]
@natoscott @tallpsmith Let me know if there's anything more I can do to assist.

**Reply by natoscott** [reply]
@pwinckles I don't know the process here, probably best to get handover from @tallpsmith if we can rouse him here.  :)

**Reply by tallpsmith** [reply]
Hey all, I'm very sorry for being underground, but lots of stuff going on with the family at the moment.  `tldr;` I will be officially retiring as of 26th September and should have more free time to help out here.  I promise to jump on this and discuss in Slack later this week.

**Reply by pwinckles** [reply]
Great, thank you! And congratulations on your impending retirement!
